alf/algorithms/sac_algorithm.py

```diff
@@ -119,23 +126,18 @@ def __init__(self,
                 networks.
             target_update_period (int): Period for soft update of the target
                 networks.
-            dqda_clipping (float): when computing the actor loss, clips the
-                gradient dqda element-wise between [-dqda_clipping, dqda_clipping].
-                Does not perform clipping if dqda_clipping == 0.
-            actor_optimizer (tf.optimizers.Optimizer): The optimizer for actor.
-            critic_optimizer (tf.optimizers.Optimizer): The optimizer for critic.
-            alpha_optimizer (tf.optimizers.Optimizer): The optimizer for alpha.
+            actor_optimizer (torch.optim.optimizer): The optimizer for actor.
+            critic_optimizer (torch.optim.optimizer): The optimizer for critic.
+            alpha_optimizer (torch.optim.optimizer): The optimizer for alpha.
             gradient_clipping (float): Norm length to clip gradients.
             debug_summaries (bool): True if debug summaries should be created.
             name (str): The name of this algorithm.
         """
-        critic_network1 = critic_network
-        critic_network2 = critic_network.copy(name='CriticNetwork2')
-        log_alpha = tfa_common.create_variable(
-            name='log_alpha',
-            initial_value=initial_log_alpha,
-            dtype=tf.float32,
-            trainable=True)
+        critic_network1 = critic_network.copy()
+        critic_network2 = critic_network.copy()
+
+        log_alpha = nn.Parameter(torch.Tensor([initial_log_alpha]))
```

Comment:
`float(initial_log_alpha)` in case that an int value is provided

Reply:
Added